### PR TITLE
Faiza

### PR DESCRIPTION
**Pull request by Faiza-tech**

### CODE REVIEW

File: src/scrabble.js [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+    Y: 4,
+    K: 5,
+    J: 8,
+    X: 8,
+    Q: 10,
+    Z: 10
+  }
+  console.log(letterValue)
+  if (!word || typeof word !== 'string') {
+    return 0
```
Comment (EwbiDev) [inline src/scrabble.js:32]: Nice stringing together multiple things into one guard clause.

File: src/scrabble.js [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+  } else {
+    let score = 0
+    const letter = word.toUpperCase().split('')
```
Comment (EwbiDev) [inline src/scrabble.js:35]: using `.split()' to make this into an array is a good idea! 🙂

File: src/scrabble.js [hunk 3]
```diff
@@ -1,5 +1,48 @@
...
+    V: 4,
+    W: 4,
+    Y: 4,
+    K: 5,
+    J: 8,
+    X: 8,
+    Q: 10,
+    Z: 10
+  }
+  console.log(letterValue)
```
Comment (EwbiDev) [inline src/scrabble.js:30]: `console.log()` is useful for when building your code for troubleshooting. But can make things a bit busy if you leave them around all the time. I would suggest getting rid of them once you're done with them.

File: src/scrabble.js [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+  if (!word || typeof word !== 'string') {
+    return 0
+  } else {
+    let score = 0
+    const letter = word.toUpperCase().split('')
+    for (let i = 0; i < letter.length; i++) {
```
Comment (EwbiDev) [inline src/scrabble.js:36]: Since `letter` is an array, you could have used an array method here instead such as `.forEach()`

**Review by EwbiDev — COMMENTED** [review]
Nice job! 😇

File: src/scrabble.js [hunk 5]
```diff
@@ -1,5 +1,48 @@
...
+      const singleLetter = letter[i]
+      if (letterValue[singleLetter]) {
+        const newScore = letterValue[singleLetter]
+        score = score + newScore
```
Comment (yee0802) [inline src/scrabble.js:40]: This could be score += newScore !

File: src/scrabble.js (hunk 4 above)
Comment (Faiza-tech) [reply src/scrabble.js:36]: Yes you are right , but i used object in the start so array method or property made a problem here so for get rid of that i used this.

File: src/scrabble.js (hunk 5 above)
Comment (Faiza-tech) [reply src/scrabble.js:40]: yes i could be

**Pull request closed without merging**